### ISSUE TITLE
Pod install failed after upgrade to 0.70.1

### ISSUE THREAD

**Issue by longb1997** [facebook/react-native#34744]
### New Version

0.70.1

### Old Version

0.66.0

### Build Target(s)

1

### Output of `react-native info`

System:
    OS: macOS 12.5
    CPU: (16) x64 Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz
    Memory: 1.04 GB / 24.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.17.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.15.0 - /usr/local/bin/npm
    Watchman: 2022.08.15.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8815526
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.16 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0 
    react-native: 0.70.1 => 0.70.1 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Issue and Reproduction Steps

Hi, I tried to upgrade my project from 0.66.0 to 0.70.1 via [React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper/), but when I tried to `pod install` then I get this error

```
[!] CocoaPods could not find compatible versions for pod "RCT-Folly":
  In Podfile:
    FBReactNativeSpec (from `../node_modules/react-native/React/FBReactNativeSpec`) was resolved to 0.70.1, which depends on
      RCT-Folly (= 2021.07.22.00)

    RCT-Folly (from `../node_modules/react-native/third-party-podspecs/RCT-Folly.podspec`)

    RNReanimated (from `../node_modules/react-native-reanimated`) was resolved to 2.2.3, which depends on
      RCT-Folly

    react-native-fbt (from `../node_modules/react-native-fbt`) was resolved to 0.0.7, which depends on
      RCT-Folly (= 2021.06.28.00-v2)
```
![Screen Shot 2022-09-21 at 14 58 41](https://user-images.githubusercontent.com/48817520/191448572-fe54e748-07c6-4048-9e95-4574e33550bb.png)

Thanks for the help!

**Comment by gagtic** [comment]
Did you try cleaning the ios/pods and ios/podfile.lock and retrying pod install?
for me sometimes 'npx pod-install' in project folder also works instead of 'pod install' in ios folder

**Comment by longb1997** [comment]
> works

yes i did, it's not working :(

**Comment by cortinico** [comment]
> react-native-fbt (from `../node_modules/react-native-fbt`) was resolved to 0.0.7, which depends on
>       RCT-Folly (= 2021.06.28.00-v2)

The issue is with `react-native-fbt` which is depending on a older version of Folly:
https://github.com/facebook/fbt/blob/d9d86a9f40af564a06c3ccdef7997aad3d4415b7/packages/react-native-fbt/react-native-fbt.podspec#L11

Please open an issue here https://github.com/facebook/fbt and ask folks on FBT to follow up
